src/components/MovieCard/MovieCard.scss

```diff
@@ -1 +1,3 @@
-// 
+.card-image img {
+  object-fit: contain;
```

Comment:
The 'object-fit: contain;' style rule ensures that the image scales while maintaining its aspect ratio, but it may not fill the entire content box of its parent element. If you want the image to cover the entire parent, you might want to use 'object-fit: cover;' instead.